proof-producer/bin/proof-producer/include/nil/proof-generator/prover.hpp

```diff
@@ -482,6 +490,19 @@ namespace nil {
                 return verification_result;
             }
 
+            bool save_circuit_to_file(boost::filesystem::path circuit_file) {
+                using namespace nil::crypto3::marshalling::types;
+
+                BOOST_LOG_TRIVIAL(info) << "Writing circuit to " <<
+                    circuit_file;
+
+                bool res = true;
```

Comment:
Looks like you forgot to write to file.

Reply:
will add after merge https://github.com/NilFoundation/placeholder/pull/111